modelscope/pipelines/cv/self_supervised_depth_completion_pipeline.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+from typing import Any, Dict
+
+from modelscope.metainfo import Pipelines
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines.base import Pipeline
+from modelscope.pipelines.builder import PIPELINES
+from modelscope.utils.constant import Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.self_supervised_depth_completion, module_name=Pipelines.self_supervised_depth_completion)
+class SelfSupervisedDepthCompletionPipeline(Pipeline):
+    """SelfSupervisedDepthCompletionPipeline Class"""
+    def __init__(self, model: str, **kwargs):
+
+        super().__init__(model=model, **kwargs)
+        logger.info('load model done')
+
+    def preprocess(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
+        """preprocess, not used at present"""
+        return inputs
+
+    def forward(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
+        """forward"""
+        model_dir = inputs['model_dir']
+        source_dir = inputs['source_dir']
+        self.model.run(model_dir, source_dir)
+        return {OutputKeys.OUTPUT: 'Done'}

```

Comment:
![image](https://github.com/modelscope/modelscope/assets/4771825/98b09c56-d105-42a3-b217-dbef5024d2b4)
as declared in outputs.py, the output of this pipeline should be an image with a key called 'output_image'